packages/cli/src/commands/translations.ts

```diff
@@ -10,10 +11,19 @@ export type TranslationsOptions = {
 
 export const handleTranslations = async ({ argv }: CommandArgs<TranslationsOptions>) => {
   process.stdout.write(`\nLaunching translate using NPX.\n\n`);
-  const npxExecutableName = process.platform === 'win32' ? 'npx.cmd' : 'npx';
-  spawn(
+  const { npxExecutableName, projectDir, locale, shell } = getPlatformArgs(argv);
+
+  const child = spawn(
     npxExecutableName,
-    ['-y', '@redocly/realm', 'translate', argv.locale, `-d=${argv['project-dir']}`],
-    { stdio: 'inherit' }
+    ['-y', '@redocly/realm', 'translate', locale ?? '', `-d=${projectDir}`],
```

Comment:
Could you move the `?? ''` into the sanitizer itself? For me it looks like it makes sense to transform all empty arguments into empty spaces.